src/widgets-service.js

```diff
@@ -1,4 +1,5 @@
 var _ = require('underscore');
+var URI = require('urijs');
```

Comment:
I udnerstand is easy to use a npm module but I'd try to avoid to use libraries for things as easy as `location.hash.slice(1)`